.kokoro/docs/docs-presubmit.cfg

```diff
@@ -15,14 +15,3 @@ env_vars: {
     key: "TRAMPOLINE_IMAGE_UPLOAD"
     value: "false"
 }
-
-env_vars: {
-    key: "TRAMPOLINE_BUILD_FILE"
-    value: "github/python-bigquery/.kokoro/build.sh"
-}
-
-# Only run this nox session.
-env_vars: {
-    key: "NOX_SESSION"
```

Comment:
@busunkim96 Is this a regression? How does the docs-presubmit build know which session(s) to run without this?

Reply:
Changes in #423 haven't made their way to the synthtool templates yet. Please ignore this PR for now. :)